### PR TITLE
project ready for marking

### PR DESCRIPTION
**Pull request by lianavaleria15**
### Work Done
- created HTML, CSS reset and styling pages
- added media queries and flexbox properties to make it responsive on different viewports 
- added pseudoclass to manipulate the size of the first project card
- used CSS variable property for colors 

### TODO
- [ ] Need to improve the code for about-me img on medium viewport size
- [ ] Encountered some issues with fixing the nabvar at the top, working on the settings
- [ ] Improve the animations once I learn JS

### CODE REVIEW

File: assets/styles/style.css [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+  --eerie-black: #161a1dff;
+  --blood-red: #660708ff;
+  --ruby-red: #a4161aff;
+  --carnelian: #ba181bff;
+  --imperial-red: #e5383bff;
+  --silver-chalice: #b1a7a6ff;
+  --light-gray: #d3d3d3ff;
+  --cultured: #f5f3f4ff;
+  --white: #ffffffff;
+}
```
Comment (LeeFarnell2U) [inline assets/styles/style.css:13]: I love root variables to this makes me really happy. Great job!

File: assets/styles/style.css [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+  list-style-type: none;
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  justify-content: flex-end;
+  padding: 2rem 2rem;
+  /* position:fixed;
+  margin-top: 0;
+  width: 100%;
+  z-index: 2; */
```
Comment (LeeFarnell2U) [inline assets/styles/style.css:37]: If things are commented out, you probably don't need them, so consider deleting them.

File: assets/styles/style.css [hunk 3]
```diff
@@ -0,0 +1,352 @@
...
+    margin: 5rem;
+    width: 25rem;
+  }
+  about-text {
+    margin-top: 15rem;
+    flex: 8 0;
+  }
+}
+
+/*keyframes for animation functions*/
```
Comment (LeeFarnell2U) [inline assets/styles/style.css:326]: Big plus for use of animation! Great work!

File: index.html [hunk 4]
```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Liana Laurentiu - My protfolio</title>
```
Comment (LeeFarnell2U) [inline index.html:7]: Please correct the spelling of Portfolio.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+
+  <body class="body">
+    <!--header containing a navigation bar with 3 links-->
+    <header class="header">
+      <nav>
+        <ul class="navlinks-container">
+          <li class="navbar-links"><a href="#about-me">About me</a></li>
+          <li class="navbar-links"><a href="#my-projects">Projects</a></li>
+          <li class="navbar-links"><a href="#my-contacts">Contact me</a></li>
+        </ul>
```
Comment (LeeFarnell2U) [inline index.html:28]: As mentioned in the feedback, I'd use capital M here for the 'Me', but I'm being really picky.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+              database theory. I am planning to use my attention to details
+              alongside creativity in building application that could solve
+              present issues.
+            </p>
+
+            <p>
+              I am also working with young people with autism, learning
+              difficulties and deafblindness. In my free time you can find me
+              reading (currently Game of Thrones, going to the gym and cooking).
+            </p>
```
Comment (LeeFarnell2U) [inline index.html:71]: As mentioned on the feedback, I think you closing bracket is in the wrong place here?

File: index.html [hunk 7]
```diff
@@ -0,0 +1,186 @@
...
+            class="contact-links"
+            >LinkedIn</a
+          >
+          <a
+            href="https://mail.google.com/"
+            target="_blank"
+            rel="noopener noreferrer"
+            class="contact-links"
+            >Email</a
+          >
```
Comment (LeeFarnell2U) [inline index.html:176]: Have a look at using the `mailto:` code here. It would work great with these.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,186 @@
...
+        </div>
+      </section>
+    </main>
+
+    <!--footer-->
+    <footer class="footer">
+      <h6>Made with coffee</h6>
```
Comment (LeeFarnell2U) [inline index.html:183]: I appreciate this on a personal level!

File: assets/styles/style.css (hunk 2 above)
Comment (lianavaleria15) [reply assets/styles/style.css]: Deleted :)

File: assets/styles/style.css (hunk 3 above)
Comment (lianavaleria15) [reply assets/styles/style.css:322]: Thank you Lee

File: index.html (hunk 5 above)
Comment (lianavaleria15) [reply index.html:28]: Corrected

File: index.html (hunk 6 above)
Comment (lianavaleria15) [reply index.html:71]: Haha, true didn't observe thank you

File: index.html (hunk 7 above)
Comment (lianavaleria15) [reply index.html:176]: Updated with mailto

**Pull request merged**